README.md

```diff
@@ -1,5 +1,369 @@
 # Substrate SSVM Node
 
+## Build
+
+To build SSVM enabled Substrate node, we need the following repositories:
+
+```
+git clone https://github.com/second-state/substrate-ssvm-node.git
+git clone https://github.com/second-state/substrate-cli-tools.git
+```
+
+and use [secondstate/ssvm](https://hub.docker.com/r/secondstate/ssvm) as building environment:
+
+```
+docker run -it --rm \
+  --name ssvm \
+  -v $(pwd):/root \
+  -w /root \
+  secondstate/ssvm:latest
+```
+
+In docker container, install rust yarn:
+
+- Install rust
+```
+(docker) curl https://sh.rustup.rs -sSf | sh -s -- -y
+(docker) source ~/.cargo/env
+(docker) rustup update nightly && rustup update stable
+(docker) rustup target add wasm32-unknown-unknown --toolchain nightly
+```
+
+- Install yarn
+```
+(docker) curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+(docker) echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
+(docker) apt update && apt install -y yarn
+```
+
+Build substrate-cli-tools:
+
+```
+(docker) cd ~/substrate-cli-tools/typescript
+(docker) yarn install && yarn run tsc
+(docker) chmod +x dist/*.js
+```
+
+Build substrate-ssvm-node:
+
+```
+(docker) cd ~/substrate-ssvm-node
+(docker) cargo build --release --verbose
+```
+
+## Run
+
+After building substrate-ssvm-node, we could start a Substrate node that supports Ewasm by our [pallet-ssvm](https://github.com/second-state/pallet-ssvm):
+
+```
+(docker) cargo run --release --bin node-template -- purge-chain --dev -y
+(docker) cargo run --release --bin node-template -- --dev
+```
+
+You could also check RPC information or listen events from pallet-ssvm:
+
+```
+# Start a new shell in the same docker container
+docker exec -it ssvm bash
+(docker) ~/substrate-cli-tools/typescript/dist/info.js
+(docker) ~/substrate-cli-tools/typescript/dist/events.js +ssvm
+```
+
+## Deploy contract
+
+Now we could use substrate-cli-tools to deploy Ewasm contract to our node. Here we use the following ERC20 contract as an example:
+
+```
```

Comment:
Attach three files and point source code sol(ERC20.sol) link url
1. ERC20.sol
2. ERC20.wasm
3. ERC20.hex

Reply:
Is it okay to put these files into `erc20/` directory at project root?